lightning/src/ln/features.rs

```diff
@@ -98,6 +98,7 @@ impl_feature_write_without_length!(Bolt12InvoiceFeatures);
 impl_feature_write_without_length!(ChannelTypeFeatures);
 impl_feature_write_without_length!(InvoiceRequestFeatures);
 impl_feature_write_without_length!(OfferFeatures);
+impl_feature_write_without_length!(BlindedHopFeatures);
```

Comment:
Had to add this line to the feature ser fix due to changes in commit `0c5922e92a3ec3fdf9226cceeb9057ad9aa9dc19`.